deps/wazuh_testing/wazuh_testing/dashboard_saturation_tests/data/tests/EndpointTest.js

```diff
@@ -0,0 +1,71 @@
+const { expect } = require("@playwright/test");
+const { PathManager } = require("./../lib/PathManager.js");
+const { CookieManager } = require("./../lib/CookieManager.js");
+const { ScreenshotManager } = require("./../lib/ScreenshotManager.js");
+
+class EndpointTest {
+
+    page = null;
+    vuContext = null;
```

Comment:
What is `vuContext`? I think it's not entirely clear, consider adding documentation to this or using a more descriptive name.

Reply:
`vuContext` contains the information that is passed to the tests. All variables in `artillery.yml` are accessible using vuContext. As for `page` it is an object with the Browser information. The documentation is [here](https://playwright.dev/docs/api/class-page)

Reply:
Solved in https://github.com/wazuh/wazuh-qa/pull/5629/commits/f46b2fc7da524084f1ae359a11bc2f0f6a02413c